client/my-sites/sharing/connections/connection.jsx

```diff
@@ -203,7 +228,7 @@ export default connect(
 		return {
 			siteId,
 			userHasCaps: canCurrentUser( state, siteId, 'edit_others_posts' ),
-			userId: getCurrentUserId( state ),
+			userId: getCurrentUserId( state )
```

Comment:
AFAIK it isn't needed. In fact I'm not sure if we should keep or not the comma here.